Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityArticulatedHand.cs

```diff
@@ -5,17 +5,23 @@
 using Microsoft.MixedReality.Toolkit.Input;
 
 #if UNITY_WSA
-using UnityEngine;
 using UnityEngine.XR.WSA.Input;
-#endif
-
-#if WINDOWS_UWP
-using Microsoft.MixedReality.Toolkit.Windows.Utilities;
+#if WINDOWS_UWP || DOTNETWINRT_PRESENT
 using System;
 using System.Collections.Generic;
+using UnityEngine;
+#if WINDOWS_UWP
+using Windows.Foundation.Metadata;
 using Windows.Perception;
 using Windows.Perception.People;
 using Windows.UI.Input.Spatial;
+#elif DOTNETWINRT_PRESENT
+using Microsoft.Windows.Foundation.Metadata;
+using Microsoft.Windows.Perception;
+using Microsoft.Windows.Perception.People;
+using Microsoft.Windows.UI.Input.Spatial;
+#endif
```

Comment:
please add comment after each #endif to help clarify which condition it is closing. thanks!

Reply:
I'm never sure which condition to put when there's an `#elif` in the middle! The original `#if` or the `#elif`?

Reply:
I would put the #elif condition